unitTests/ui/QuerySuggestAddonTest.ts

```diff
@@ -188,5 +188,45 @@ export function QuerySuggestAddonTest() {
         done();
       });
     });
+
+    describe('when the characterThresholdForSuggestions option is set to a value', () => {
+      let magicBoxText;
+      let spyOnEvent;
+      beforeEach(() => {
+        simulateFakeSuggestions({ completions: [{ executableConfidence: 1, expression: 'foo', highlighted: 'foo', score: 1 }] });
+        magicBoxText = jasmine.createSpy('magicBoxText');
+        spyOnEvent = jasmine.createSpy('spyOnEvent');
```

Comment:
maybe rename the variable to something more meaningful like `querySuggestSuccessSpy`

Reply:
Will do